components/conjunctions/_screen-sm-min.scss

```diff
@@ -13,14 +13,14 @@
 		flex-direction: row;
 
 		&:not(.brdr-0) {
-			[class*="cnjnctn-col"] {
+			> [class*="cnjnctn-col"] {
 				min-height: 3em;
 				padding-left: 0;
 				padding-right: 0;
 			}
 		}
 
-		[class*="cnjnctn-col"]:not(:first-child) {
+		> [class*="cnjnctn-col"]:not(:first-child) {
```

Comment:
You have > [class*="cnjnctn-col"] twice. They could be combined and then have &:not(:first-child) and &:not(:last-child) as subrules.